GVRf/Framework/backend_oculus/src/main/java/org/gearvrf/OvrXMLParser.java

```diff
@@ -106,6 +106,10 @@
                                 settings.setUseGazeCursorController(Boolean
                                         .parseBoolean(xpp.getAttributeValue(i)));
                             } else if (attributeName
+                                    .equals("useAndroidWearTouchpad")) {
```

Comment:
added the check to the day dream parser and verified by running the gvr-video sample on a day dream phone.